simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -0,0 +1,81 @@
+            buildGauge("jetty_request_time_max", "maximum time spend handling requests (in ms)", statisticsHandler.getRequestTimeMax()),
+            buildGauge("jetty_request_time_total", "total time spend in all request handling (in ms)", statisticsHandler.getRequestTimeTotal()),
+            buildGauge("jetty_request_time_mean", "mean time spent handling requests (in ms)", statisticsHandler.getRequestTimeMean()),
+            buildGauge("jetty_request_time_std_dev", "standard deviation for request handling (in ms)", statisticsHandler.getRequestTimeStdDev()),
+            buildGauge("jetty_dispatched", "number of dispatches", statisticsHandler.getDispatched()),
+            buildGauge("jetty_dispatched_active", "number of dispatches currently active", statisticsHandler.getDispatchedActive()),
+            buildGauge("jetty_dispatched_active_max", "maximum number of active dispatches being handled", statisticsHandler.getDispatchedActiveMax()),
+            buildGauge("jetty_dispatched_time_max", "maximum time spend in dispatch handling", statisticsHandler.getDispatchedTimeMax()),
+            buildGauge("jetty_dispatched_time_total", "total time spent in dispatch handling (in ms)", statisticsHandler.getDispatchedTimeTotal()),
+            buildGauge("jetty_dispatched_time_mean", "mean time spent in dispatch handling (in ms)", statisticsHandler.getDispatchedTimeMean()),
+            buildGauge("jetty_dispatched_time_std_dev", "standard deviation for dispatch handling (in ms)", statisticsHandler.getDispatchedTimeStdDev()),
+            buildGauge("jetty_async_requests", "total number of async requests", statisticsHandler.getAsyncRequests()),
+            buildGauge("jetty_async_requests_waiting", "currently waiting async requests", statisticsHandler.getAsyncRequestsWaiting()),
+            buildGauge("jetty_async_requests_waiting_max", "maximum number of waiting async requests", statisticsHandler.getAsyncRequestsWaitingMax()),
+            buildGauge("jetty_async_dispatches", "number of requested that have been asynchronously dispatched", statisticsHandler.getAsyncDispatches()),
+            buildGauge("jetty_expires", "number of async requests requests that have expired", statisticsHandler.getExpires()),
+            buildStatusGauge(),
+            buildGauge("jetty_stats_on_ms", "time in milliseconds stats have been collected for", statisticsHandler.getStatsOnMs()),
+            buildGauge("jetty_responses_bytes_total", "total number of bytes across all responses", statisticsHandler.getResponsesBytesTotal())
+    );
+  }
+
+  private static MetricFamilySamples buildGauge(String name, String help, double value) {
+    return new MetricFamilySamples(
+            name,
+            Type.GAUGE,
+            help,
+            Collections.singletonList(new MetricFamilySamples.Sample(name, EMPTY_LIST, EMPTY_LIST, value)));
+  }
+
+  private MetricFamilySamples buildStatusGauge() {
+    String name = "jetty_responses";
```

Comment:
It's used at https://github.com/prometheus/client_java/pull/120/files#diff-f47a35564451e6d56e9b2ec018b94324R65